packages/schemas/src/types/log.ts

```diff
@@ -96,6 +96,15 @@ interface SignInSocialLogPayload extends SignInSocialBindLogPayload {
   redirectTo?: string;
 }
 
+export type IssuedTokenType = 'accessToken' | 'refreshToken' | 'idToken';
```

Comment:
same here, is there a way to limit it by the oidc-provider types?

Reply:
There are some related constants in oidc provider, but there are no related enums and etc.

Reply:
Maybe I can do it in this way, but it seems not to be a good idea.
![image](https://user-images.githubusercontent.com/10594507/168787610-43ec88f1-d514-40f3-b3e9-dccaf633c5e7.png)

Any better idea?

Reply:
keep it for now.